CodeStarterCamp_Week4/routine.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  routine.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by MARY on 2023/03/16.
+//
+
+import Foundation
+
+enum RoutineError: Error {
+    case exhaust
+    case invalidInput
+    case outOfRangeInput
+}
+
+struct Routine {
+    let name: String
+    let activities: [Activity]
+    let numberInKorean: [String] = ["첫", "두", "세", "네", "다섯", "여섯", "일곱", "여덟", "아홉", "열"]
+         
+    func start(conditions: BodyCondition) throws {
+        let sets = try getNumberOfSets()
+        
+        for set in 1...sets {
+            print("\(numberInKorean[set-1]) 번째 \(name)을(를) 시작합니다.")
+            try workOut(conditions: conditions)
+        }
+    }
+    
+    private func getNumberOfSets() throws -> Int {
+        print("루틴을 몇 번 반복할까요?")
+        guard let sets = Int(readLine()!) else {
+            throw RoutineError.invalidInput
+        }
+        
+        guard isInRange(sets: sets) else {
+            throw RoutineError.outOfRangeInput
+        }
+        
+        print("--------------")
+        
+        return sets
+    }
+    
+    private func isInRange(sets: Int) -> Bool {
+        sets > 0 && sets <= 10 ? true : false
```

Comment:
삼항 연산자를 활용하셨군요! 👍👍
한가지 팁을 더 드리자면, 조건의 결과 자체가 Bool타입이라서 요렇게도 쓸 수 있습니다.
가독성은 좀 떨어지지만, 코드는 더 간단해져서 상황에 따라선 활용할 수도 있어요. 😁
```suggestion
        sets > 0 && sets <= 10
```